src/seedu/addressbook/data/person/Email.java

```diff
@@ -58,4 +60,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableString(Printable... printables) {
+		return PREFIX_EMAIL + value;
```

Comment:
Notice that your indentation mechanism uses tabs for indentation. 
The coding standards we follow require using 4 spaces for indentation.
Tip: you can set Eclipse to do this for you, check out how!